Automattic/wp-calypso
# Reader Recent Feed Overhaul: Add mobile view

Related to https://github.com/Automattic/loop/issues/193

## Proposed Changes

* Add responsive styling and column separation.
* Open posts full screen on mobile screen sizes.
* Some cleanup to prevent a console warning.
* **No go: adding a back button from the full post.**


https://github.com/user-attachments/assets/5323dc4d-2853-4aa8-82db-ae03ad4665cf



## Why are these changes being made?


* See pe7F0s-2iv-p2

## Testing Instructions



* Go to /read?flags=reader/recent-feed-overhaul
* View at different screen sizes.
* Click into a post on desktop and it should open on the right.
* Click into a post on mobile and it should open full screen.

Note, there's no back button from the full post yet.

## Pre-merge Checklist
